projects/storefrontstyles/scss/components/cds/carousel/_merchandising-carousel.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../product/carousel/product-carousel';
+
+%cx-merchandising-carousel {
+  @extend %cx-product-carousel;
+  background-color: var(
```

Comment:
I would not mind to reuse the existing `--cx-color-background` and `--cx-color-text` CSS variables will get more specific for the selector you use and all inherited content.